frontend/templates/layouts/default.j2

```diff
@@ -1,3 +1,14 @@
+{% macro filter(formats='websites, ads, stories, email') -%}
+{% if not formats.strip() %}
+{{ caller() }}
+{% else %}
+{% set formats = formats.split(',') %}
+<p class="{% for format in formats %}ap--{{ format.strip() }} {% endfor %}">
```

Comment:
Should this be a `<div>` instead? `<p>` could produce side effects, as they come with margins, and nested elements could be weird..

Reply:
Had a `div` before but Grow didn't add `p`s then which *made* it look weird 😁I'll test again...

Reply:
`\n` to the rescue. Now `div` is the wrapping element and Grow gracefully adds `p` - too many as always, but... 🙂